### PR TITLE
python39Packages.ruamel: adopt

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-burns** [comment]
Ping for eval error, didn't see what exactly was causing it

Hyphenated names look good to me, and appear to be predominant on repology :+1:

File: pkgs/top-level/python-aliases.nix [hunk 1]
```diff
@@ -79,6 +79,10 @@ mapAliases ({
   qasm2image = throw "qasm2image is no longer maintained (since November 2018), and is not compatible with the latest pythonPackages.qiskit versions."; # added 2020-12-09
   requests_toolbelt = requests-toolbelt; # added 2017-09-26
   rotate-backups = throw "rotate-backups was removed in favor of the top-level rotate-backups"; # added 2021-07-01
+  ruamel_base = ruamel-base; # added 2021-11-01
+  ruamel_ordereddict = ruamel-ordereddict; # added 2021-11-01
```
Comment (jonringer) [inline pkgs/top-level/python-aliases.nix:83]: This seems to only exist in the python2 package set.

Comment (SuperSandro2000) [reply pkgs/top-level/python-aliases.nix:83]: It does. I am not sure wether we should add an alias for it but if it causes an eval error the answer is no.

**Comment by SuperSandro2000** [comment]
> Hyphenated names look good to me, and appear to be predominant on repology 👍

and this follows PEP something which is written in the python language docs for nixpkgs.

**Comment by vcunat** [comment]
Some `ruamel_yaml` occurrences remained.  That now causes a problem, as some places are strict in not allowing aliases, e.g. it would become a channel blocker: https://hydra.nixos.org/build/157517452

**Comment by SuperSandro2000** [comment]
I was in the wrong directory when I did the replacement. Fixing that right now.

**Pull request merged**